src/lib/datasource/InMemoryDatasource.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 const Datasource = require('../Datasource');
 
-class InMemoryDatasource extends Datasource {
+
+const undefinedToNull = (value) => 
+    value === undefined ? null : value;
+
+class InMemoryDatasource extends Datasource{
```

Comment:
Please, follow space emplacement convension

Reply:
It will be in a new file with normal space implementation.